packages/trackerless-network/test/integration/NetworkNode.test.ts

```diff
@@ -0,0 +1,99 @@
+import { NetworkNode } from '../../src/NetworkNode'
+import { NodeType, PeerDescriptor, Simulator, SimulatorTransport } from '@streamr/dht'
+import {
+    MessageID,
+    MessageRef,
+    StreamMessage,
+    StreamMessageType,
+    StreamPartIDUtils,
+    toStreamID
+} from '@streamr/protocol'
+import { EthereumAddress, waitForCondition } from '@streamr/utils'
+
+describe('NetworkNode', () => {
+
+    let transport1: SimulatorTransport
+    let transport2: SimulatorTransport
+
+    let node1: NetworkNode
+    let node2: NetworkNode
+
+    const pd1: PeerDescriptor = {
+        kademliaId: new Uint8Array([1, 2, 3]),
```

Comment:
Is there some significance about these numbers (vs. numbers at line 27) in this test? Would it make sense to create a helper function which generates a random kademliaId if the test should work with any id?